server.go

```diff
@@ -157,7 +157,10 @@ func (s *Server) stopWorkers() {
 		w.stop()
 	}
 
-	s.workers = nil
+	// fix nil exception
+	s.workers = []*worker{}
```

Comment:
`s.workers` is a map, not a array, maybe i need to add CI.

Reply:
I'm sorry, I have not pull right branch